core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -34,7 +35,8 @@ angular.module('oppia').directive('imageUploader', [
         height: '@',
         onFileChanged: '=',
         errorMessage: '@',
-        width: '@'
+        width: '@',
+        allowedImageFormats: '@'
```

Comment:
Seems like you're not passing a literal, so do '&' instead.

getAllowedImageFormats: '&allowedImageFormats'

Reply:
Done.